libcommon/sources/UnixPlatform.cxx

```diff
@@ -0,0 +1,51 @@
+#include "pivot/Platform.hxx"
+
+#ifdef PLATFORM_UNIX
+
+    #include <fcntl.h>
+    #include <string.h>
+    #include <sys/utsname.h>
+    #include <unistd.h>
+
+    #include <unordered_map>
+
+namespace pivot
+{
+
+bool plateform::isDebuggerPresent()
```

Comment:
Could be a good idea to check once on startup and cache the result. Otherwise, we're going to get really bad performances.

Reply:
Can be because a compiler can be attached while pivot is already running.